package/MDAnalysis/transformations/nojump.py

```diff
@@ -123,6 +123,17 @@ def _transform(self, ts):
         except np.linalg.LinAlgError:
             msg = f"Periodic box dimensions are not invertible at step {ts.frame}"
             raise NoDataError(msg)
```

Comment:
Somehow I'm glad I'm not the only one that totally used `NoDataError` the wrong way around for ages - no need to fix here but we should consider fixing this later as part of #3901

Reply:
Yeah the first usage a few lines before looks ok, but this is arguably ValueError or something, including the values of the dimensions would also help debug (I'm assuming if it's not invertible it's all zeros or something)

Reply:
Yeah I've definitely misappropriated `NoDataError` outside of MDAnalysis too in other code I've written...